ia-guide/ia-static.md

```diff
@@ -0,0 +1,341 @@
+If you have already made a start on the docs, that's great!
+But even if you think you're starting from scratch, there's a reasonable chance you actually already have something.
+
+Have a think about it:
+
+* Have you written notes while you've been working?
+    Even if they're just scratches on a notebook on your desk, that's documentation!
+* Have you written any comments in your code?
+    You really should have! They're documentation, too.
+* Have you put a README in your code repository?
+    That's also documentation!
+* Have you given any talks or presentations about your project?
+    Even if there isn't a recording, any notes or slides you made is documentation too!
+
+Get all of these things together and look at them with your critical paths in mind.
+Do you have content here that fits all those critical paths?
+
+You will probably start to notice the gaps pretty quickly.
+It could be that you have a whole lot of content that describes how to do it, but not a lot that explains what it is.
+Or, you could have a lot of stuff aimed at beginners, and not a lot that works for experts.
+Sometimes, you might find that you have a lot of descriptions of things, but not a lot of step-by-step instructions.
+
+## What's missing?
+
+Have a think about each of your critical paths, and identify what content you need to fulfil each one.
+
+At this stage, it's normal to be feeling a little overwhelmed.
+Now you can work out what you actually need, so you can cut some things from the list.
+
+## State your intent
+
+If you wanted to build a house, you wouldn't start laying bricks until you'd had a think about what kind of house you want to build.
+How big is it going to be?
+What materials are you going to use?
+What layout and design is it going to have?
+
+You also need to think about what you intend for your documentation.
+Is it going to be comprehensive and detailed, or simple and sparse?
+Is it going to use simple language, designed for people who have never done this before, or more technical language, for people who know what they're doing and just need the basics?
+Is it going to start from the very beginning, or are you going to assume that your readers already have some knowledge?
+
+## Face reality
+
+By now, you should have a good idea of what your ideal documentation looks like.
+It will be a shining beacon of docs goodness, hitting all the critical paths for all your readers, and people will talk about it for years to come!
+
+It's a grim reality that project documentation needs to be maintained.
+Have a think about how many writers your project might be able to access.
+Are you likely to have writers on the project over time?
+What skill level are they likely to have?
+Is there any content that already exists elsewhere that you can point to, rather than writing it yourself?
+Is there any way you can automate any part of the documentation, such as using code comments to generate API documentation?
+
+## Choose a direction
```

Comment:
Think this is good and I'd maybe group it into a larger __Planning__ piece to work with __Measure the distance__